onboard/nav/searcher.hpp

```diff
@@ -42,6 +41,13 @@ class Searcher {
     NavState executeDriveToBall( Rover* phoebe );
 
 protected:  // TODO
+    /*************************************************************************/
+    /* Protected Member Functions */
+    /*************************************************************************/
+
+    //function for correcting distances between points
+    void correctPoints( Rover * phoebe, const rapidjson::Document& roverConfig ); 
```

Comment:
I'm not sure this function name captures what your function does as you don't actually modify any points. Something more like `insertIntermediatePoints` might be more accurate and help others understand your function better when they see it called.